IVCollectionKit/Source/CollectionSection.swift

```diff
@@ -81,7 +81,9 @@ open class CollectionSection : AbstractCollectionSection {
     }
     
     open func sizeForItem(at indexPath: IndexPath, boundingSize: CGSize) -> CGSize {
-        return items[indexPath.item].estimatedSize(boundingSize: boundingSize)
+        let sizeWithInsets = CGSize(width: boundingSize.width - insetForSection.left - insetForSection.right,
```

Comment:
I can't totally agree with this change. For instance, we might not wanna use vertical insets for CollectionView with vertical scrolling. For some cases we need to know exact size of collection view and only use horizontal / vertical section insets for proportions calculation. I made change in 1.0.0 branch where we can respect section insets and spacings in cell size calculation. If u wanna match ur cell width to collectionView width and section insets u should consider use `adjustsWidth(true)` modifier of `CollectionItem`

Reply:
see this example https://github.com/ivedeneev/CollectionKit/blob/master/Examples/ViewController.swift line 29